sqlmesh/dbt/loader.py

```diff
@@ -48,9 +49,13 @@ def sqlmesh_config(
     if variables is not None:
         loader_kwargs["variables"] = variables
 
+    connection_kwargs = kwargs.pop("connection_kwargs", {})
+    if register_comments is not None:
+        connection_kwargs["register_comments"] = register_comments
```

Comment:
Where do `kwargs` get the `connection_kwargs` from? I'm trying to wrap my head around the data flow.